app/uk/gov/hmrc/accessibilitystatementfrontend/tasks/MilestoneReportTask.scala

```diff
@@ -32,15 +32,21 @@ class MilestoneReportTask @Inject() (
   )
 
   override def getBodyRows: Seq[Seq[String]] = for {
-    statement <- accessibilityStatementRepo.findAll if isEnglishStatement(statement)
+    statement <- accessibilityStatementRepo.findAll
+    if isEnglishStatement(statement) && isPublicStatement(statement)
     row       <- getWcagCriterion(statement) ++ getUnmatchedMilestones(statement)
   } yield getMilestoneCells(row)
 
-  private def isEnglishStatement(statementTuple: (String, Lang, AccessibilityStatement)) = {
+  private def isEnglishStatement(statementTuple: (_, Lang, _)) = {
```

Comment:
It's not a massive deal, but this code feels a bit fragile with the whole tuple being passed through like this with type undefined. It would be cleaner if we either:
1. Decomposed the tuple before passing through just the lang to the function
2. We could use a type for `StatementTuple` to cut down on boilerplate